tests/data/aos7.json

```diff
@@ -78125,519 +78125,2135 @@
+                    "remote_version": "",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/15",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/15",
+                    "ifName": "2/1/15"
+                },
+                {
+                    "active": 1,
+                    "protocol": "lldp",
+                    "remote_hostname": "rox-lugoj-19",
+                    "remote_port": "4050 (2cfaa2575a8c)",
+                    "remote_platform": null,
+                    "remote_version": "",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/19",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/19",
+                    "ifName": "2/1/19"
+                },
+                {
+                    "active": 1,
+                    "protocol": "lldp",
+                    "remote_hostname": "roahroddtimi05.hro.hella.com",
+                    "remote_port": "Gi0 (00fec8206430)",
+                    "remote_platform": null,
+                    "remote_version": "Cisco IOS Software, C2700 Software (AP3G2-K9W8-M), Version 15.3(3)JF10, RELEASE SOFTWARE (fc3)\nTechnical Support: http://www.cisco.com/techsupport\nCopyright (c) 1986-2019 by Cisco Systems, Inc.\nCompiled Thu 13-Jun-19 18:51 by prod_rel_team",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/21",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/21",
+                    "ifName": "2/1/21"
+                },
+                {
+                    "active": 1,
+                    "protocol": "lldp",
+                    "remote_hostname": "rox-lugoj-23",
+                    "remote_port": "1049 (e8e7325f9ff8)",
+                    "remote_platform": null,
+                    "remote_version": "Alcatel-Lucent Enterprise OS6450-P48 6.7.2.122.R08 GA, September 04, 2020.",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/23",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/23",
+                    "ifName": "2/1/23"
+                },
+                {
+                    "active": 1,
+                    "protocol": "lldp",
+                    "remote_hostname": "rox-lugoj-24",
+                    "remote_port": "1049 (2cfaa2202a18)",
+                    "remote_platform": null,
+                    "remote_version": "",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/24",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/24",
+                    "ifName": "2/1/24"
+                },
+                {
+                    "active": 1,
+                    "protocol": "lldp",
+                    "remote_hostname": "rox-lugoj-26",
+                    "remote_port": "2049 (2cfaa25759bc)",
+                    "remote_platform": null,
+                    "remote_version": "",
+                    "ifAlias": "Alcatel-Lucent OS6860 GNI 2/1/26",
+                    "ifDescr": "Alcatel-Lucent OS6860 GNI 2/1/26",
```

Comment:
Are we sure about these ? it seems that we have a lot more LLDP entries, and some are redundant.

Reply:
hi @PipoCanaja 
spun off AOS7 snmpsim, and observed the resulting DB

it is 200++ port device, big one
and there is total 68 links detected on AOS7 test data

observing DB, there is no duplicate !!!
every 
`local_port_id + remote_hostname + remote_port` 
combination is uniq !

simple visual exploring of 'LLDP-MIB::lldpRemTable' will show that there is much much more entry than in old test.json
have no idea why old test.json was so small

looked at the old test data, every entry from old test is included in new discovery data
so, at least this part is OK